ontopy/ontology.py

```diff
@@ -1223,6 +1226,8 @@ def new_entity(self, name: str, parent: ThingClass) -> ThingClass:
                 f"Error in label name definition {name}: "
                 "Label consists of more than one word."
             )
-        with self:
-            entity = types.new_class(name, (parent,))
+        parent = parent if isinstance(parent, list) else [parent]
```

Comment:
Better to test against `collections.abc.Sequence` than just `list` to accept any list-like type, like tuples.

Reply:
Maybe you can call the new variable `parents` to emphasise that it is a sequence. Then you even avoid modifying the argument, which is considered good practice.

Reply:
testing against a collections.abc.Iterable now, and renamed to parents.